packages/mco/components/dr-status-popover/parsers/dr-parser-utils.tsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import { VolumeReplicationHealth } from '@odf/mco/constants';
+import { useDisasterRecoveryResourceWatch } from '@odf/mco/hooks';
+import {
+  getDRClusterResourceObj,
+  getDRPlacementControlResourceObj,
+  getDRPolicyResourceObj,
+} from '@odf/mco/hooks';
+import { Progression } from '@odf/mco/types';
+import { findDRType, getReplicationHealth } from '@odf/mco/utils';
+import {
+  ClusterVersionConditionType,
+  getResourceCondition,
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+  YellowExclamationTriangleIcon,
+} from '@odf/shared';
+import { formatTime } from '@odf/shared/details-page/datetime';
+import { InProgressIcon } from '@patternfly/react-icons';
+
+export const useDRResources = (namespace: string) => {
+  return useDisasterRecoveryResourceWatch({
+    resources: {
+      drClusters: getDRClusterResourceObj(),
+      drPolicies: getDRPolicyResourceObj(),
+      drPlacementControls: getDRPlacementControlResourceObj({ namespace }),
+    },
+  });
+};
```

Comment:
```suggestion

};
```
No need to create one more hook on top of useDisasterRecoveryResourceWatch, call useDisasterRecoveryResourceWatch from parser directly.